src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/benificialowner/IndividualBeneficialOwnerChange.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner.psc.IndividualBeneficialOwnerPsc;
+
+public class IndividualBeneficialOwnerChange extends BeneficialOwnerChange<IndividualBeneficialOwnerPsc> {
+  // No additional fields, use getters and setters from abstract class
```

Comment:
The comment say No additional fields, - confusing as but there are. Remove comment?

Reply:
Sure